bigquery_etl/parse_udf.py

```diff
@@ -41,12 +41,31 @@ def from_file(filepath):
         with open(filepath) as f:
             text = f.read()
 
+        name = basename.replace(".sql", "")
```

Comment:
@relud, might be good for you to review this section. I may have manged `parse_udf` in order to generate some SQL UDF tests.